### PR TITLE
shellcheck fixes

### PR DESCRIPTION
**Pull request by nickboldt**
Change-Id:...

shellcheck fixes

Change-Id: Ife6ab5696d70334b28af498f8f9c806899a7b3cc
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: .ci/cico-happy-path-pr-test.sh [hunk 1]
```diff
@@ -65,6 +65,7 @@ install_deps
 
 # Build & push.
 
+# shellcheck disable=SC2154
```
Comment (benoitf) [inline .ci/cico-happy-path-pr-test.sh:68]: hello, why we disable check ?

File: .ci/cico_common.sh [hunk 2]
```diff
@@ -68,6 +68,8 @@ function installAndStartMinishift() {
   oc adm policy add-cluster-role-to-user cluster-admin developer
   oc login -u developer -p developer
 
+  # shellcheck source=che-cert_generation.sh
+  # shellcheck disable=SC1091
```
Comment (benoitf) [inline .ci/cico_common.sh:72]: hello, why we disable check ?

Comment (nickboldt) [reply .ci/cico_common.sh:72]: because we know where the file is and we don't need to worry about it moving suddenly.

Comment (nickboldt) [reply .ci/cico_common.sh:72]: and every attempt I did to make this check happy failed. so... https://github.com/koalaman/shellcheck/wiki/SC1091 decided to disable it.

File: .ci/cico_common.sh [hunk 3]
```diff
@@ -124,9 +126,10 @@ function obtainUserToken() {
...
-  TEST_USERNAME=admin
-  TEST_PASSWORD=admin
+  TEST_USERNAME="admin"
+  TEST_PASSWORD="admin"
 
-  export USER_ACCESS_TOKEN=$(curl -k -v -X POST $KEYCLOAK_BASE_URL/realms/che/protocol/openid-connect/token -H "Content-Type: application/x-www-form-urlencoded" -d "username=${TEST_USERNAME}" -d "password=${TEST_PASSWORD}" -d "grant_type=password" -d "client_id=che-public" | jq -r .access_token)
+  USER_ACCESS_TOKEN="$(curl -k -v -X POST "$KEYCLOAK_BASE_URL"/realms/che/protocol/openid-connect/token -H "Content-Type: application/x-www-form-urlencoded" -d "username=${TEST_USERNAME}" -d "password=${TEST_PASSWORD}" -d "grant_type=password" -d "client_id=che-public" | jq -r .access_token)"
+  export USER_ACCESS_TOKEN
   echo "========User Access Token: $USER_ACCESS_TOKEN "
 }
```
Comment (ericwill) [inline .ci/cico_common.sh:135]: Missing newline

File: .ci/cico-happy-path-pr-test.sh [hunk 4]
```diff
@@ -102,7 +103,7 @@ else
     getOpenshiftLogs
     archiveArtifacts "che-plugin-registry-prcheck"
     echo "Failed while starting chectl. Please check logs available in http://artifacts.ci.centos.org/devtools/che/che-plugin-registry-prcheck/${ghprbPullId}/"
-    exit 1337
+    exit 133
```
Comment (ericwill) [inline .ci/cico-happy-path-pr-test.sh:106]: This may be a dumb question, but what's the significance of this change?

Comment (nickboldt) [reply .ci/cico-happy-path-pr-test.sh:106]: Shellcheck says you can't return a number greater than 255.

So assuming whoever wrote this wanted a value other than 1 I figured I'd change it to a less "leet" value that was kinda "lee".

Comment (ericwill) [reply .ci/cico-happy-path-pr-test.sh:106]: Sounds fair to me!

**Comment by amisevsk** [comment]
PR https://github.com/eclipse/che-plugin-registry/pull/481 has been open for 6 days

**Comment by ericwill** [comment]
I merged #481, so I think this one can be closed now right @nickboldt ?

File: .ci/cico_common.sh (hunk 3 above)
Comment (nickboldt) [reply .ci/cico_common.sh:135]: after line 135?

**Comment by nickboldt** [comment]
replaced by https://github.com/eclipse/che-plugin-registry/pull/491

**Pull request closed without merging**